### ISSUE TITLE
Show a special icon for pages that are signed which don't have rules set

### ISSUE THREAD

**Issue by tasn** [tasn/webext-signed-pages#3]
This is somehow related to issue #1 in making this extension even easier to use.

The idea is that if you have the extension installed, and you visit a website that signs it pages, but you don't have configured, it'll show you an icon indicating of that and suggesting you configure the website. With potentially automatic configuration (a sort of trust on first use) if you just click on the icon.

**Comment by danielrozenberg** [comment]
Trust on first use is not unreasonable. Maybe eventually in the future some websites would like to be hard coded into the extension or into some trusted database that the user can opt out of, but we're not there yet

**Comment by tasn** [comment]
I agree. Especially since it's TOFU anyhow because you have to get the configuration from somewhere. One needs a safe (external?) way to verify anyway.

Hardcoded: I agree, this is already on my mental to-do.

Thanks for your input.
